r2/r2/models/subreddit.py

```diff
@@ -553,11 +580,25 @@ def remove_moderator(self, user, **kwargs):
     def moderators(self):
         return self.moderator_ids()
 
+    @property
+    def top_moderator_account(self):
```

Comment:
Let's just drop both of these properties too, I think. We can just check the IDs directly above, so we don't really need `top_moderator_account` or `top_moderator`.